roles/database/tasks/main.yml

```diff
@@ -8,17 +8,15 @@
     postgresql_locales:
       - 'es_ES.UTF-8'
 
-- import_role:
+- include_role:
     name: geerlingguy.postgresql
-    tasks_from: users
   vars:
     postgresql_users:
       - name: "{{ database_user }}"
         role_attr_flags: "{{ database_role_attributes }}"
 
-- import_role:
+- include_role:
     name: geerlingguy.postgresql
-    tasks_from: databases
```

Comment:
If you remove this the whole role get executed, we just need `databases` tasks to be executed, same above for `users`.